setup.py

```diff
@@ -39,8 +39,8 @@
     ],
     packages=find_packages(exclude=["tests"]),
     package_data={"aio_pika": ["py.typed"]},
-    install_requires=["aiormq~=6.2.2", "yarl"],
-    python_requires=">=3.6, <4",
+    install_requires=["aiormq~=6.2.3", "yarl"],
+    python_requires=">3.6, <4",
```

Comment:
This reverts https://github.com/mosquito/aio-pika/pull/442

I'd appreciate if you change it back to `python_requires=">=3.6, <4"` and release a version of aio-pika including that change.